octoprint_authldap/templates/authldap_settings.jinja2

```diff
@@ -0,0 +1,79 @@
+<form class="form-horizontal">
+    <h3>LDAP configuration</h3>
+    <div class="control-group">
+        <h4>General</h4>
+        <label class="control-label">{{ _('LDAP URI') }}</label>
+        <div class="controls">
+            <input type="text" class="input-block-level" data-bind="value: settings.plugins.authldap.ldap_uri"
+                   placeholder="ldaps://ldap.server.com">
+        </div>
+
+        <label class="control-label">{{ _('Search base pattern') }}</label>
+        <div class="controls">
+            <input type="text" class="input-block-level" data-bind="value: settings.plugins.authldap.ldap_search_base"
+                   placeholder="CN=Users,DC=example,DC=com">
+        </div>
+
+        <label class="control-label">{{ _('LDAP Filter') }}</label>
+        <div class="controls">
+            <input type="text" class="input-block-level" data-bind="value: settings.plugins.authldap.ldap_query"
+                   placeholder="(&(objectclass=user)(memberof=<GROUPNAME>)(samaccountname={uid}))">
+        </div>
+
+        <label class="control-label">{{ _('Default Roles:') }}</label>
+        <div class="controls">
+            <input type="text" class="input-block-level" data-bind="value: settings.plugins.authldap.roles"
+                   placeholder="Comma separated list of roles: user,admin">
+        </div>
+        <div class="controls">
+            <label class="checkbox">
+                <input type="checkbox" data-bind="checked: settings.plugins.authldap.auto_activate">Automatically activate users?
+            </label>
+        </div>
+    </div>
+    <div class="control-group">
+        <h4>Authentication</h4>
+        <label class="control-label">{{ _('Binding User DN') }}</label>
+        <div class="controls">
+            <input type="text" class="input-block-level" data-bind="value: settings.plugins.authldap.ldap_bind_user"
```

Comment:
This is the service user, which is used for querying the LDAP. Do not confuse it with the user trying to login later. 
Once OctoPrint has a more granular user access control, you could also query for groups, emails, etc.